### DISCUSSION TITLE
UI is transparent and almost everytime when I press the live button i get a black preview (even not responding) CUDA is installed

### DISCUSSION THREAD

**Discussion by aalextudoran** [hacksider/Deep-Live-Cam#654] [General]
![image](https://github.com/user-attachments/assets/a1f54d1f-aed0-49d3-b060-dd9d0255cba7)

**Comment by KRSHH** [comment]
Download this in venv -

1. `.\venv\Scripts\Activate`
2. ` python -m pip install --force-reinstall https://github.com/KRSHH/FileStorage/raw/refs/heads/main/customtkinter-5.2.2-py3-none-any.whl`

**Comment by KRSHH** [comment]
Please tell, if it fixes it or not.

**Reply by aalextudoran** [reply]
![image](https://github.com/user-attachments/assets/10435254-4a1d-48ea-a935-05ed4ebf4f01)


I have in Scripts the activate.bat and I run it before starting the actual application. I also installed what you provided, however after starting the app, I have this error: 
..\...\Programs\Python\Python310\lib\tkinter\__init__.py", line 2383, in __getattr__ return getattr(self.tk, attr)
AttributeError: '_tkinter.tkapp' object has no attribute 'block_update_dimensions_event'

**Reply by aalextudoran** [reply]
I'm using python 3.10, cuda 11.8, onnx 1.16 and cudNN 8.5.0.96

**Reply by aalextudoran** [reply]
Update: an workaround is to remove the call for block_update_dimensions_event from the scaling_tracker.py file, but I still get a black preview when I press Live

**Reply by barrywass** [reply]
Any solution my friend? I'm having the same problem

**Reply by barrywass** [reply]
The fix above worked for me, the reinstallation thank you so much!

**Reply by KRSHH** [reply]
Great!
